src/components/Dashboard.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react'
 
 import {
+  AccountPage,
   Password,
+  Sign,
   Transactions,
   Transfer,
   WalletInfo,
 } from '@containers'
+import { AccountPage as AccountPageEnum } from '@types'
 
-export const Dashboard: React.SFC = () => (
-  <div className='vertical-spaced'>
-    <div className='has-text-centered title-heading is-constant-height'>
-      <Password />
-    </div>
+export interface Props {
+  accountPage: AccountPageEnum
+}
+
+export const TransferPage: React.SFC = () => (
```

Comment:
I'd also make this `Dashboard` or `AccountDashboard`

Reply:
And split it into a separate file.

Reply:
yep done